cpmpy/expressions/core.py

```diff
@@ -550,13 +550,22 @@ def get_bounds(self):
             lbs, ubs = zip(*[get_bounds(x) for x in self.args])
             return sum(lbs), sum(ubs)
         elif self.name == 'wsum':
-            lbs, ubs = zip(*[get_bounds(x) for x in self.args])
-            return sum(self.args[0] * np.array(lbs)), \
-                   sum(self.args[0] * np.array(ubs))
+
```

Comment:
We had a pretty clean and fast one for this case: https://github.com/CPMpy/cpmpy/blob/b75f577006f598ec0220a0a81b55461a9336be39/cpmpy/transformations/flatten_model.py#L336
Maybe better to re-use that one?
Also, would it be usefull to have a helper function "get_bounds_args" that returns the bounds of an expressions arguments?